.github/workflows/ci.yaml

```diff
@@ -81,6 +81,14 @@ jobs:
       - uses: actions/checkout@v4
       - run: cargo fmt --all -- --check
 
+  cargo-deny:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: EmbarkStudios/cargo-deny-action@v1
+      with:
+        arguments: --workspace
```

Comment:
Good point, updated to add `--all-features`. It doesn't make any difference right now because we don't have any deps controlled by features (https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies), but if we add any in the future it will be good to have this job do the right thing already.